packages/i18n/utils.js

```diff
@@ -1,15 +1,18 @@
-// TODO: remove once you have more than one
-/* eslint-disable import/prefer-default-export */
+export const extractLanguageTagFromLocale = locale =>
+  locale.includes('-') ? locale.split('-')[0] : locale;
 
-export const getMatchingMomentCode = lang => {
-  switch (lang) {
-    case 'en':
-      return 'en-gb';
-    case 'de':
-      return 'de';
-    case 'es':
-      return 'es';
-    default:
-      return 'en-gb';
-  }
+export const mapLocaleToMomentLocale = locale => {
+  if (locale.startsWith('de')) return 'de';
+  if (locale.startsWith('es')) return 'es';
+  if (locale.startsWith('fr')) return 'fr';
+  if (locale === 'zh-CN') return 'zh-cn';
+  return 'en-gb';
```

Comment:
😕 why is the default for moment different than for intl?

Reply:
That's how their data files are defined 🤷‍♂️